CMSgov/design-system
# [WNMGDS-1340] Replace use of @extend for util classes in Medicare SCSS

## Summary

https://jira.cms.gov/browse/WNMGDS-1340

This removes the use of `@extend` in the `ds-medicare-gov` styles for utility classes. They added a lot of unnecessary `!important`s to the component styles.

Here are some examples of this sort of work being done for the core design system: 
- https://github.com/CMSgov/design-system/pull/1186
- https://github.com/CMSgov/design-system/pull/1182
- https://github.com/CMSgov/design-system/pull/1179
- https://github.com/CMSgov/design-system/pull/1175

TO DO:
- [x] Rebase this branch after https://github.com/CMSgov/design-system/pull/1679 has been merged

### Changed

- We no longer use `@extend` in `ds-medicare-gov` to apply utility classes to component styles. The following components have been updated:
  - `NavBar`
  - `NavigationMenu`
  - `SimpleFooter`
  - `Card`
  - Note that we are not updating the deprecated `GlobalHeader` component

## How to test

- Run the visual regression tests for medicare (`yarn storybook:medicare` in one terminal and `LOKI_FILE_PREFIX=medicare yarn loki test` in another)
- Play around with the three affected components and make sure there's nothing weird
- You can manually compare your local storybook in the browser with [a storybook built prior to these changes](http://design-system-demo.s3-website-us-east-1.amazonaws.com/pwolfert/vrt-child-design-systems/mgov/storybook/) if you wish.
